elementary/cli/cli.py

```diff
@@ -45,6 +45,7 @@ def format_help(self, ctx, formatter):
         self.format_epilog(ctx, formatter)
 
     def invoke(self, ctx: click.Context) -> Any:
+        set_root_logger_file_handler("elementary")
```

Comment:
I think it may be good that `set_root_logger_file_handler` will explicitly get the target path rather than relying on the click context.
WDYT about moving the call to `config.py` below and then doing `set_root_logger_file_handler("elementary", target_path)`?

I think it makes sense to do it right after you're creating the directory there.

Reply:
Hmm I see at that point in the code the config still points to the default edr_files path, maybe you can show me why 
but I can also get it from the click context in the cli.py instead of the log (unless what you dislike here is the context itself and not the location of it)